src/VisualStudio/Core/Def/Implementation/EditAndContinue/VisualStudioActiveStatementProvider.cs

```diff
@@ -48,11 +46,7 @@ void CancelWork()
                         FreeBuilders(builders);
                         builders = null;
 
-                        // TODO: DkmWorkList.Cancel doesn't currently work when invoked on the completion callback.
-                        // We continue execute all the queued callbacks -- they will be no-ops.
-                        // See https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems/edit/562781.
-                        // 
-                        // workList.Cancel();
+                        workList.Cancel();
```

Comment:
has this been tested out?

Reply:
I believe this should be Cancel(false)

Reply:
Microsoft.VisualStudio.Debugger.DkmWorkList

public void Cancel();

I could not see any other overloading of the method.

---
In reply to: [198277822](https://github.com/dotnet/roslyn/pull/28070#discussion_r198277822) [](ancestors = 198277822)

Reply:
Waiting for a fix in VS. The current build 15.8.0 Preview 4.0 [27902.1.d15.8] still has the same issue. 

It is promised that the fix is available in D15.8stg 27827.3003 

---
In reply to: [198275263](https://github.com/dotnet/roslyn/pull/28070#discussion_r198275263) [](ancestors = 198275263)

Reply:
Verified with 15.8 preview 5.

---
In reply to: [200483266](https://github.com/dotnet/roslyn/pull/28070#discussion_r200483266) [](ancestors = 200483266,198275263)